frontend/packages/helm-plugin/src/components/list-page/ProjectHelmChartRepositoryListPage.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { ListPage } from '@console/internal/components/factory';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ProjectHelmChartRepositoryModel } from '../../models';
+import ProjectHelmChartRepositoryList from './ProjectHelmChartRepositoryList';
+
+const ProjectHelmChartRepositoryPage: React.FC<React.ComponentProps<typeof ListPage>> = (props) => {
+  const createProps = {
+    to: `/ns/${props.namespace || 'default'}/projecthelmchartrepositories/~new/`,
```

Comment:
Isn't it just ~/new without a trailing slash for other resources?

```suggestion
    to: `/ns/${props.namespace || 'default'}/projecthelmchartrepositories/~new`,
```